apps/tools/tests/test_real_courses.py

```diff
@@ -53,7 +46,7 @@ def test_error_1(tmpdir):
 
 def test_error_2(tmpdir):
     course = load("e2")
-    settings = Settings(
+    settings = fakes.Settings(
```

Comment:
Hi @jeremykenny this PR is great :tada: but there's just one problem. Looks like this `fakes.Settings` is different, and does not have `dry_run`, and therefore these tests fail:

you can see that the CI fails because of this: https://kantord.semaphoreci.com/jobs/68410e8f-b9a1-4876-ae26-6d1a45df489d

you can also see the failures on your machine if you run the following command:

```bash
poetry run pytest
```

Reply:
Please fix it in a way that the tests don't fail. If you get stuck, [I can jump on a call with you and help out](https://app.harmonizely.com/harmonizely-uoicqk/15min).

Reply:
Hi @kantord my apologies for the delay. I've figured it out, no call needed, but thank you for the support. I believe the CI will pass now.